script.js

```diff
@@ -1 +1,189 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+/*
+  checkboxNumber: todoCard가 unique한 id를 갖기 위한 변수
+*/
+let checkboxNumber = 0;
+
+//todoCard 만들기
+const createTodoCard = (todo, isDone) => {
+  const todoCard = document.createElement('div');
+  const checkbox = document.createElement('input');
+  const todoContent = document.createElement('label');
+  const todoDelete = document.createElement('div');
+
+  todoCard.className = 'todoCard';
+  checkbox.type = 'checkbox';
+  checkbox.className = 'todoCheckbox';
+  checkbox.id = `todoCheckbox${checkboxNumber}`;
+  if (isDone) {
+    checkbox.checked = true;
+    todoCard.className = 'doneCard';
+  } else {
+    todoCard.className = 'todoCard';
+  }
+  todoContent.className = 'todoContent';
+  todoContent.htmlFor = `todoCheckbox${checkboxNumber++}`;
+  todoContent.textContent = todo;
+  todoDelete.className = 'todoDelete';
+  todoDelete.textContent = 'X';
+
+  todoCard.append(checkbox, todoContent, todoDelete);
+  if (isDone) {
+    document.querySelector('.doneSection').append(todoCard);
+  } else {
+    document.querySelector('.todoSection').append(todoCard);
+  }
+};
+
+//localstorage에 저장된 todo,done 불러오기
+const todoList = JSON.parse(localStorage.getItem('todo'));
+const donelist = JSON.parse(localStorage.getItem('done'));
+if (todoList) {
+  todoList.forEach((todo) => {
+    createTodoCard(todo, false);
+  });
+}
+if (donelist) {
+  donelist.forEach((todo) => {
+    createTodoCard(todo, true);
+  });
+}
+
+//localstorage에 todo,done 추가하기
+const addTodoToLocalStorage = (todo) => {
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+  localStorage.removeItem('todo');
+  if (todoList) {
+    const newTodoList = [...todoList, todo];
+    localStorage.setItem('todo', JSON.stringify(newTodoList));
+  } else {
+    localStorage.setItem('todo', JSON.stringify([todo]));
+  }
+};
+const addDoneToLocalStorage = (todo) => {
+  const doneList = JSON.parse(localStorage.getItem('done'));
+  localStorage.removeItem('done');
+  if (doneList) {
+    const newDoneList = [...doneList, todo];
+    localStorage.setItem('done', JSON.stringify(newDoneList));
+  } else {
+    localStorage.setItem('done', JSON.stringify([todo]));
+  }
+};
+
+//localstorage에 todo,done 삭제하기
+const deleteTodoFromLocalStorage = (todo) => {
+  const todoList = JSON.parse(localStorage.getItem('todo'));
+  const newTodoList = todoList.filter((item) => item !== todo);
+  localStorage.setItem('todo', JSON.stringify(newTodoList));
+};
+const deleteDoneFromLocalStorage = (todo) => {
+  const doneList = JSON.parse(localStorage.getItem('done'));
+  const newDoneList = doneList.filter((item) => item !== todo);
+  localStorage.setItem('done', JSON.stringify(newDoneList));
+};
+
```

Comment:
`deleteTodoFromLocalStorage` 와 `deleteDoneFromLocalStorage` 로직에 겹치는 부분이 많은 것 같아서 다음과 같이 함수를 하나로 합쳐도 좋을 것 같습니다:)
```suggestion
const deleteFunc = (todo, listName) => {
   const itemList = JSON.parse(localStorage.getItem(listName));
   const updatedList = itemList.filter((item) => item !== todo);
   localStorage.setItem(listName, JSON.stringify(updatedList));
};
```